src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/RecognisePriorLearningRequestToDetailsViewModelMapper.cs

```diff
@@ -1,37 +1,109 @@
 ﻿using SFA.DAS.CommitmentsV2.Api.Client;
 using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.ProviderCommitments.Interfaces;
 using SFA.DAS.ProviderCommitments.Web.Models;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.ProviderCommitments.Web.Mappers.Apprentice
 {
+    public class RecognisePriorLearningRequestToDataViewModelMapper : IMapper<RecognisePriorLearningRequest, PriorLearningDataViewModel>
+    {
+        private readonly IOuterApiService _outerApiService;
+
+        public RecognisePriorLearningRequestToDataViewModelMapper(IOuterApiService outerApiService)
+        {
+            _outerApiService = outerApiService;
+        }
+
+        public async Task<PriorLearningDataViewModel> Map(RecognisePriorLearningRequest source)
+        {
+            var result = new PriorLearningDataViewModel();
```

Comment:
Trivial comment, but personally I would move this line to below the outer api call so that the first set of properties can be initialized at the point of construction. Neither here nor there really though.

Reply:
moved